about.html

```diff
@@ -17,24 +17,24 @@
 
     <main>
       <h1>Title of About Page</h1>
-      <section>
+      <section id="LearnMoreButton">
```

Comment:
Follow the conception of naming and use the "kebab-case" for the current id name "learn-more-button"